bq-metrics-extractor/src/main/java/org/ostelco/bqmetrics/BqMetricsExtractorApplication.kt

```diff
@@ -0,0 +1,192 @@
+package org.ostelco.bqmetrics
+
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.google.cloud.bigquery.*
+import io.dropwizard.Application
+import io.dropwizard.setup.Bootstrap
+import io.dropwizard.setup.Environment
+import io.prometheus.client.exporter.PushGateway
+import io.prometheus.client.CollectorRegistry
+import io.dropwizard.Configuration
+import io.dropwizard.cli.ConfiguredCommand
+import io.prometheus.client.Summary
+import net.sourceforge.argparse4j.inf.Namespace
+import net.sourceforge.argparse4j.inf.Subparser
+import org.slf4j.LoggerFactory
+import java.util.*
+import org.slf4j.Logger
+import javax.validation.Valid
+import javax.validation.constraints.NotNull
+
+
```

Comment:
Add a proper "class comment" to this file stating its intent and overall architecture.